cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations.pt

```diff
@@ -531,12 +538,6 @@ policy "aws_ri_recommendations" do
       field "instanceClass" do
         label "Instance Class"
       end
-      field "instanceSize" do
-        label "Instance Size"
-      end
-      field "instanceType" do
-        label "Instance Type"
-      end
       field "licenseModel" do
         label "License Model"
       end
```

Comment:
It looks like we need to rename the next field to lookBackPeriod

Reply:
Done!